post_info.php

```diff
@@ -132,6 +140,21 @@ function execute_sql($query_string, $bind_variables, $operation)
     }
 }
 
+/**
+ *  Convert "false", "0" to false, Convert others to true
+ * @param variable:bool/string/int The variable that we want to convert
+ * @return bool
+ */
+
+function conversion_to_bool($variable)
```

Comment:
`as_bool` is shorter. Why don't use use the stackoverflow method? Is there a downsite to it? I'm sure it catches more cases than this.